Soccar [Game-Server]/src/nl/soccar/gameserver/util/MapUtilities.java

```diff
@@ -9,16 +9,23 @@
 import org.jbox2d.dynamics.World;
 
 /**
+ * Utilites for the Map.
+ *
  * @author PTS34A
  */
 public final class MapUtilities {
 
     private static final float WALL_WIDTH = 5;
-    private static final float CORNER_SIZE = 15;
 
     private MapUtilities() {
     }
 
+    /**
+     * Adds wall to the Map.
+     *
+     * @param engine The given game engine, not null.
+     * @param map The given map, not null.
+     */
     public static void addWalls(GameEngine engine, Map map) {
         Rectangle size = map.getSize();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)